src/navigation/RepetitionStackNavigator.tsx

```diff
@@ -0,0 +1,26 @@
+import { createStackNavigator } from '@react-navigation/stack'
+import React, { ReactElement } from 'react'
+
+import RepetitionScreen from '../routes/repetition/RepetitionScreen'
+import { getLabels } from '../services/helpers'
+import { RoutesParams } from './NavigationTypes'
+import screenOptions, { useTabletHeaderHeight } from './screenOptions'
+
+const Stack = createStackNavigator<RoutesParams>()
+
+const RepetitionStackNavigator = (): ReactElement | null => {
+  const options = screenOptions(useTabletHeaderHeight())
+  const { back } = getLabels().general
+
+  return (
+    <Stack.Navigator>
+      <Stack.Screen
+        name='Repetition'
+        component={RepetitionScreen}
+        options={({ navigation }) => options(back, navigation)}
+      />
```

Comment:
If we just have one screen inside the navigator, it should not be necessary to wrap it in another StackNavigator. Or is it planned to add more screens here later on?

Reply:
I don't think so. Check out the design on Figma: https://www.figma.com/file/4bW33nU6zZ6ySeTqrM37Jq/Lunes_Master?type=design&node-id=301-2481&mode=design&t=D20ByDd0SMUTdbAB-0

Reply:
@sarahsporck  should I better comment out the tab on navigation?

Reply:
No that's not neccessary. If even, we'll do this in a separate branch. I think @steffenkleinle wanted to point out, that as this route does not have any more screens. Therefore, you can directly put the `RepetitionScreen` into the `RepetitionTab` in the `BottomTabNavigator`, then you could get rid of the `RepetitonStackNavigator`.

Reply:
I think we should comment out the tab, otherwise we can't release anymore bc this screen is not yet finished.